### PR TITLE
Adds example for GameObject property bindings

### PR DESCRIPTION
**Pull request by JochenHeckl**

### CODE REVIEW

File: Runtime/Scripts/GameObjectPropertyBindings.cs
Comment (benedikt-peter) [inline Runtime/Scripts/GameObjectPropertyBindings.cs:1]: I'd suggest to add another property here for the GameObject's name.

**Review by benedikt-peter — CHANGES_REQUESTED** [review]

File: Runtime/Scripts/GameObjectPropertyBindings.cs
Comment (JochenHeckl) [reply Runtime/Scripts/GameObjectPropertyBindings.cs:1]: My bad. I added name as we discussed

Comment (benedikt-peter) [reply Runtime/Scripts/GameObjectPropertyBindings.cs:1]: I think it should be string instead of int.

Comment (JochenHeckl) [reply Runtime/Scripts/GameObjectPropertyBindings.cs:1]: and you are right!

**Review by benedikt-peter — APPROVED** [review]

**Review by benedikt-peter — APPROVED** [review]

**Pull request merged**